ccd-definition/ComplexTypes/DQ/RequestedCourt.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "RequestedCourt",
+    "ListElementCode": "claimantRequestedCourt",
+    "FieldType": "Label",
+    "ElementLabel": "### Court name \n ${courtLocation.applicantPreferredCourt}",
```

Comment:
did we not change this already to applicant1PreferredCourt? I'm sure this was one of the refactored items

Reply:
if it was meant to be changed it hasn't been. Please raise a JIRA ticket, I am not going to fix as a part of 562